troubleshooting.md

```diff
@@ -0,0 +1,56 @@
+![PODMAN logo](logo/podman-logo-source.svg)
+
+# Troubleshooting
+
+## A list of common issues and solutions for Podman
+
+---
+### 1) No such image or Bare keys cannot contain ':'
+
+When doing a `podman pull` or `podman build` command and a "common" image can not be pulled,
+it is likely that the `/etc/containers/registries.conf` file is either not installed or possibly
+misconfigured.
+
+#### Symptom
+```
+sudo podman build -f Dockerfile 
+STEP 1: FROM alpine
+error building: error creating build container: no such image "alpine" in registry: image not known
+```
+or
+```
+$ sudo podman pull fedora
+error pulling image "fedora": unable to pull fedora: error getting default registries to try: Near line 9 (last key parsed ''): Bare keys cannot contain ':'.
+```
+
+#### Solution
+
+  * Verify that the `/etc/containers/registries.conf` file exists.  If not, verify that the skopeo-containers package is installed.
+  * Verify that the entries in the `[registries.search]` section of the /etc/containers/registries file are valid and reachable.
+    *  i.e. `registries = ['registry.fedoraproject.org', 'quay.io', 'registry.access.redhat.com']`
+
+---
+### 2) http: server gave HTTP response to HTTPS client
+
+When doing a Podman command such as `build`, `commit`, `from`, or `push` to a registry,
```

Comment:
We don't have a `podman from`, but I think `podman pull` can trigger this

Reply:
darn cut/paste will touch that up and check on the travis complaints.